scene/2d/camera_2d.cpp

```diff
@@ -527,11 +544,9 @@ void Camera2D::align() {
 
 void Camera2D::set_follow_smoothing(float p_speed) {
 
+	// enable / disable should now be called explicitly
+	// rather than relying on this function to enable / disable
```

Comment:
That comment makes sense in the context of this diff but not that of the code after the change :)

Reply:
Ok will change this.